### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ereslibre**

### CODE REVIEW

**Comment by ereslibre** [comment]
It works. However the deploy fails because we didn't migrate https://github.com/kubic-project/velum/pull/104 to https://github.com/kubic-project/caasp-container-manifests. I can take a look at it.

**Comment by ereslibre** [comment]
https://github.com/kubic-project/terraform/commit/00515174d73a96a174bd35be1207703542412a8a introduced a problem in which we never really wait for the port to be there.

However, and when https://github.com/kubic-project/velum/pull/104/commits/5224276eef306de95d9323e5202098548ff4309b hits https://github.com/kubic-project/caasp-container-manifests we won't need to wait for a port but for a unix socket (just to keep it in mind).

This is why the `until` isn't working as expected on `provision-dashboard.sh`:
```
$ netstat -lnt | awk '$6 == "LISTEN" && $4 ~ ".THISDOESNTEXIST"'
$ echo $?
0
```

File: provision/dashboard/provision-dashboard.sh [hunk 1]
```diff
@@ -49,8 +49,8 @@ ETCD_REPL="s|#\?ETCD_LISTEN_PEER_URLS.*|ETCD_LISTEN_PEER_URLS=http://0.0.0.0:238
 MANIF_PATHS_SUBS="s|/usr/share/salt/kubernetes/pillar|/tmp/salt/pillar|g; \
                   s|/usr/share/salt/kubernetes/salt|/tmp/salt/sls|g; \
                   s|/usr/share/caasp-container-manifests/config/salt/grains/ca|/tmp/salt/grains/ca|g; \
-                  s|/usr/share/caasp-container-manifests/config/salt/minion.d-ca/|/tmp/salt/sls/ca/|g; \
-                  s|/usr/share/caasp-container-manifests/config/salt/minion.d-ca/|/tmp/salt/config/minion.d-ca/|g; \
+                  s|/usr/share/caasp-container-manifests/config/salt/minion.d-ca/minion.conf|/tmp/salt/config/minion.d-ca/minion.conf|g; \
+                  s|/usr/share/caasp-container-manifests/config/salt/minion.d-ca/signing_policies.conf|/tmp/salt/sls/ca/signing_policies.conf|g; \
```
Comment (inercia) [inline provision/dashboard/provision-dashboard.sh]: I would keep out the things that do not change in the replacements: the filenames `minion.conf` and `signing_policies.conf`.

Comment (ereslibre) [inline provision/dashboard/provision-dashboard.sh]: That's exactly what the fix does :) They share the same root on the source, but they don't share the same root on the target.

`/usr/share/caasp-container-manifests/config/salt/minion.d-ca/minion.conf` -> `/tmp/salt/config/minion.d-ca`
`/usr/share/caasp-container-manifests/config/salt/minion.d-ca/signing_policies.conf` -> `/tmp/salt/sls/ca`

So there's no way to do a substitution based only on `/usr/share/caasp-container-manifests/config/salt/minion.d-ca`, it wouldn't work.

Another solution would be to move the files around on terraform, so they match 1:1 the location that caasp-container-manifests has.

Comment (inercia) [inline provision/dashboard/provision-dashboard.sh]: I understand, you are right, we must use the full paths...

**Pull request merged**